index.html

```diff
@@ -67,25 +68,9 @@
         </div>
     </main>
     <main id="about">
+        <h4 class="title">A Little Bit About Me</h4>
         <div class="about-me-area">
-            <h4 class="about-me-title">A Little Bit About Me...</h4>
-            <p class="about-me-text">I said a hip hop
-                Hippie to the hippie
-                The hip, hip a hop, and you don't stop, a rock it out
-                Bubba to the bang bang boogie, boobie to the boogie
-                To the rhythm of the boogie the beat
-                Now, what you hear is not a test I'm rappin' to the beat
-                And me, the groove, and my friends are gonna try to move your feet
-                See, I am Wonder Mike, and I'd like to say hello
-                To the black, to the white, the red and the brown
-                The purple and yellow, but first, I gotta
-                Bang bang, the boogie to the boogie
-                Say up jump the boogie to the bang bang boogie
-                Let's rock, you don't stop
-                Rock the rhythm that'll make your body rock
-                Well so far you've heard my voice but I brought two friends along
-                And the next on the mic is my man Hank
-                C'mon, Hank, sing that song, check it out</p>
+            <p class="about-me-text">I’m currently developing my skills as a Full Stack developer.<br><br> Prior to this I have had a diverse work background including being part of the Product Owner team for Cancer Research UK’s flagship product, teaching English in Tokyo and helping a small Cornish company create an app for their staff.<br><br>Outside of studying I enjoy keeping fit and being outdoors. I am looking forward to competing at the Turf Games in London this summer and driving my VW van to Cornwall to go paddle-boarding. </p>
```

Comment:
I would ditch the <br> tags and instead just add each paragraph to it's own <p> tag. Also, you probably don't need the 'about-me-text' class, and could target the p's using '.about-me-area>p'

Reply:
going to skip due to time constraints
:)